src/mds/ScrubStack.cc

```diff
@@ -281,8 +280,12 @@ bool ScrubStack::validate_inode_auth(CInode *in)
   }
 }
 
-void ScrubStack::scrub_dir_inode(CInode *in, bool *added_children, bool *done)
+void ScrubStack::scrub_dir_inode(CInode *in, bool *added_children, bool *done, bool is_eval_stray)
```

Comment:
IMO no need to add the `bool is_eval_stray` parameter, you can get this from the `header` below.

Reply:
yeah this was something i was experimenting with, i had removed this long back. thanks.